packages/uikit/src/widgets/RoiCalculator/AssetCard.tsx

```diff
@@ -107,6 +107,7 @@ export const AssetCard = memo(function AssetCard({
       value={value}
       amount={amount}
       showPrice={showPrice}
+      decimals={18}
```

Comment:
use `currency.decimals` ?

Reply:
it's the decimal to usd not the currency itself so I think we can give it a default value